src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-your-applications-kubernetes.mdx

```diff
@@ -329,69 +305,72 @@ Follow these troubleshooting tips as needed.
      If there are no new entries on the logs, it means that the apiserver is not being able to communicate with the webhook service, this could be due to networking rules or security groups rejecting the communication.
   4. To check if the apiserver is not being able to communicate with the webhook you should inspect the apiserver logs for errors like:
 
-     ```
+     ```shell
      failed calling webhook "metadata-injection.newrelic.com": <var>ERROR_REASON</var>
      ```
 
      To get the apiserver logs:
 
      1. Start a proxy to the Kubernetes API server by the executing the following command in a terminal window and keep it running.
 
-        ```
+        ```shell
         kubectl proxy --port=8001
         ```
      2. Create a new pod in your cluster, this will make the apiserver try to communicate with the webhook. The following command will create a busybox.
 
-        ```
+        ```shell
         kubectl create -f https://git.io/vPieo
         ```
      3. Retrieve the apiserver logs.
 
-        ```
+        ```shell
         curl localhost:8001/logs/kube-apiserver.log > apiserver.log
         ```
      4. Delete the busybox container.
 
-        ```
+        ```shell
         kubectl delete -f https://git.io/vPieo
         ```
      5. Inspect the logs for errors.
 
-        ```
+        ```shell
         grep -E 'failed calling webhook' apiserver.log
         ```
-
-     Remember that one of the [requirements for the metadata injection](#network-req) is that the apiserver must be allowed egress to the pods running on the cluster. If you encounter errors regarding connection timeouts or failed connections, make sure to check the security groups and firewall rules of the cluster.
+      <Callout variant="tip">
+         One of the [requirements for the metadata injection](#network-req) is that the apiserver must be allowed egress to the pods running on the cluster. If you encounter errors regarding connection timeouts or failed connections, make sure to check the security groups and firewall rules of the cluster.
```

Comment:
#network-req? Is it pointing to something?

Reply:
Yes, it should bne pointing to `Compatibility and requirements > Network requirements`

Reply:
resolved